archivist/errors.py

```diff
@@ -34,8 +34,8 @@ class ArchivistUnwithdrawnError(ArchivistError):
     """Sbom failed to be withdrawn after fixed timeout"""
 
 
-class ArchivistIllegalArgumentError(ArchivistError):
```

Comment:
Why was this removed? Don't we need both?

Reply:
Turned out that IllegalArgument was not used anywhere. I would assume that this may change whe dataclasses are introduced